docs/setup/bare_metal.md

```diff
@@ -41,14 +43,36 @@ _The parentheses above forces the commands to be executed in a subshell, and ret
 
 ### 1.3 Configure environment variables
 
-Many configurations have default values pre-defined. If your environment requires different values, run `export ENV=value` to set environment variables in the current session (or add them to whatever profile file you are using).
+#### Required variables
+
+You will need to generate and configure `EWALLET_SECRET_KEY` and `LOCAL_LEDGER_SECRET_KEY` which are used for encrypting data in the `ewallet_db` and `local_ledger_db` respectively:
```

Comment:
Wouldn't it make sense to have default values for those in development / testing?

Reply:
You are right. These keys should already be defaulted on dev environment.